components/itemDetailsScene.xml

```diff
@@ -0,0 +1,19 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<component name="itemDetailsScreen" extends="Scene" initialFocus="ButtonGroup">
```

Comment:
Usually, we name components with a capital letter. So instead of `itemDetailsScreen`, it should be `ItemDetailsScreen`. This means that you need to change the XML file name too.

Reply:
Got it